registration/src/main/proto/spine/samples/lobby/conference/conference_service.proto

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright 2015, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package spine.samples.lobby.conference;
+
+option java_package="org.spine3.samples.lobby.conference";
+option java_outer_classname = "ConferenceServiceProto";
+option java_multiple_files = true;
+
+import "spine/samples/lobby/common/common.proto";
+//import "google/protobuf/timestamp.proto";
+
+// The request to reserve seats at the conference.
+// Is sent as the result of a registrant interaction with the UI.
+
+service ConferenceService{
+    rpc createConference(ConferenceInfo) returns (CreateConferenceResponse){}
+//    rpc updateConference(EditableConferenceInfo) returns (UpdateConferenceResponse){}
+//    rpc publish(ConferenceId) returns (UpdateConferenceResponse){}
+
+
+}
+
+message ConferenceInfo{
+    // The unique conference id.
+//    spine.samples.lobby.common.ConferenceId id = 1;
+
+    // The conference name. Used in the slug.
+    string name = 2;
+
+    // The conference description.
+    string description = 3;
+
+    // The location of the conference.
+    string location = 4;
+
+    // The part of the URL used to access the conference.
+//    spine.samples.lobby.common.ConferenceSlug slug = 5;
+
+    // The tagline of the conference.
+    string tagline = 6;
+
+    // The search query for Twitter.
+    string twitter_search = 7;
+
+    // The start date of the conference.
+//    spine.time.LocalDate start_date = 8;
+
+    // The end date of the conference.
+//    spine.time.LocalDate end_date = 9;
+
+    // The types of seats available for the conference.
+//    repeated spine.samples.lobby.common.SeatType seat_type = 10;
+
+    // The info about the owner of the conference.
+//    spine.samples.lobby.common.PersonalInfo owner = 11;
+
+    // If the conference is published it is visible for users.
+    bool is_published = 12;
+}
+message CreateConferenceResponse{
+    string id = 1;
+}
```

Comment:
Also, don't forget an empty line at the end of the file (github complains on that).